### PR TITLE
[CHK-2976] fix: set `additionalPaymentInformations` timestamps to naive date time format

### PR DESCRIPTION
**Pull request by giovanniberti**
#### List of Changes

 * changed timestamps populated inside `closePayment` `additionalPaymentInformation` like cards, now the populated timestamp is an ISO-8601 local date-time with `Europe/Rome`/`Europe/Paris` timezone

#### Motivation and Context

Wrong date-time format broke the  flow after `closePayment`

#### How Has This Been Tested?

Updated unit tests + tested with `pagopa-ecommerce-local`

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Depends on pagopa/pagopa-infra#2205

### CODE REVIEW

File: src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt [hunk 1]
```diff
@@ -1182,8 +1183,10 @@ class NodeServiceTests {
       val expectedTimestamp =
         OffsetDateTime.parse(
             authCompletedEvent.data.timestampOperation, DateTimeFormatter.ISO_OFFSET_DATE_TIME)
+          .atZoneSameInstant(ZoneId.of("Europe/Paris"))
```
Comment (pietro-tota) [inline src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt:1186]: please parameterize those tests with expected timestamp as done [here](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/0173a8ac15f439a3a90012647da9deb560fa6e75/src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt#L226-L228). This take into account both the expected string output format (making it hardcoded into tests) and Italy timezone variation (you can re-use the same method source as done per this test)

**Review by pietro-tota — APPROVED** [review]
Well done! just a trivial issue :D

File: src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt (hunk 1 above)
Comment (giovanniberti) [reply src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt]: Improved tests in 662ad21

**Review by pietro-tota — DISMISSED** [review]
🚀

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**